torch/quantization/quantize.py

```diff
@@ -85,10 +62,10 @@ def propagate_qconfig_(module, qconfig_dict=None):
 
     Args:
         module: input module
-        qconfig_dict: dictionary that maps from name or type of submodule to quantization
-            configuration, qconfig applies to all submodules of a given
-            module unless qconfig for the submodules are specified (when the
-            submodule already has qconfig attribute)
+        qconfig_dict: dictionary that maps from name or type of submodule to
```

Comment:
btw, do you have a python formatting tool

Reply:
No -- doing it manually LOL. It was just irritating to see it go off screen :)